packages/moshicam/src/abi.ts

```diff
@@ -0,0 +1,20 @@
+export const IMOSHI_PIC1155_ABI = [
+  {
+    type: 'function',
+    inputs: [
+      { name: 'to', internalType: 'address', type: 'address' },
+      { name: 'id', internalType: 'uint256', type: 'uint256' },
+      { name: 'quantity', internalType: 'uint256', type: 'uint256' },
+    ],
+    name: 'collect',
+    outputs: [],
+    stateMutability: 'payable',
+  },
+  {
+    type: 'function',
+    inputs: [],
+    name: 'mintPrice',
+    outputs: [{ name: '', internalType: 'uint256', type: 'uint256' }],
+    stateMutability: 'view',
+  },
+] as const
```

Comment:
Could you remove this `as const` assertion. Its causing a type error on the simulate contract return. https://github.com/gallery-so/questdk-plugins/blob/1ec752a0d360d4ba65b4b428dec4c04d5bfadf88/packages/moshicam/src/Moshicam.ts#L132

Reply:
Ahh ty this was what was causing the build error, I was scratching my head figuring this one out.